BlueEastCode/bluerain-os
# Feature/debugger registry

Issue:

## What I did
Add Debugger Registry

## How to test
Not now

Is this testable with jest or storyshots?
Not now

Are any of the existing tests are failing ?
Tests are passing but have to add new tests.

Does this need an update to the documentation?
Yes

If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
